joomla/joomla-cms
# [com_modules] Complete search tooltip

#### Summary of Changes

Simple PR to improves the search tooltip and element label (accessibility) in com_modules.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Extensions -> Modules, check the tooltip in the search field.
3. Go to frontend edit, click "+Module" in TinyMce and check the tooltip in the search field of the popup.
